packages/cfd/src/Containers/mt5-desktop-redirect-option.tsx

```diff
@@ -0,0 +1,88 @@
+
+type TMT5DesktopRedirectOptionProps = {
+    account_title: string;
+    mt5_trade_account: DetailsOfEachMT5Loginid;
+};
+
+const MT5DesktopRedirectOption = ({ account_title, mt5_trade_account }: TMT5DesktopRedirectOptionProps) => {
+    const desktopDownloadOptions = getDownloadOptions({ mt5_trade_account });
+
+    return (
+        <Fragment>
+            <div className='cfd-trade-modal__download-center-app'>
+                {desktopDownloadOptions.map(option => (
+                    <div key={option.icon} className='cfd-trade-modal__download-center-app--option'>
+                        <Icon icon={option.icon} size={32} />
+                        <Text className='cfd-trade-modal__download-center-app--option-item' size='xs'>
+                            {option.text}
+                        </Text>
+                        <a
+                            className='cfd-trade-modal__download-center-app--option-link'
+                            type='button'
+                            href={option.href as string}
+                            target='_blank'
+                            rel='noopener noreferrer'
+                        >
+                            <Text size='xxs' weight='bold' color='prominent'>
+                                {option.button_text}
+                            </Text>
+                        </a>
+                    </div>
+                ))}
+            </div>
+            <Text align='center' as='div' className='cfd-trade-modal__download-center-text' size='xxs' weight='bold'>
+                {localize(
+                    'Download {{ platform }} on your phone to trade with the {{ platform }} {{ account }} account',
+                    {
+                        platform: getCFDPlatformLabel(CFD_PLATFORMS.MT5),
+                        account: account_title,
+                    }
+                )}
+            </Text>
+            <div className='cfd-trade-modal__download-center-options'>
+                <div className='cfd-trade-modal__download-center-options--mobile-links'>
+                    <a
+                        href={mt5_trade_account?.white_label?.download_links?.ios}
+                        target='_blank'
+                        rel='noopener noreferrer'
+                    >
+                        <Icon icon='IcInstallationApple' width={135} height={40} />
+                    </a>
+                    <a
+                        href={mt5_trade_account?.white_label?.download_links?.android}
+                        target='_blank'
+                        rel='noopener noreferrer'
+                    >
+                        <Icon icon='IcInstallationGoogle' width={135} height={40} />
+                    </a>
+                    <a href={getPlatformMt5DownloadLink('huawei')} target='_blank' rel='noopener noreferrer'>
+                        <Icon icon='IcInstallationHuawei' width={135} height={40} />
+                    </a>
```

Comment:
Can we map this instead of repeating this ? 👀

Reply:
ok bosku